### PR TITLE
Documentation Update

### PR DESCRIPTION
**Pull request by ralgozino**
This PR updates the various readmes to the new standard for Fury 1.23. There's some improvement left to do in each package readme, until we decide what to do with them

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,152 @@
...
+```yaml
+bases:
+  - name: opa/gatekeeper
+    version: "v1.6.0"
+```
+
+> See `furyctl` [documentation][furyctl-repo] for additional details about `Furyfile.yml` format.
+
+1. Execute `furyctl vendor -H` to download the packages
+
```
Comment (nikever) [inline README.md:73]: The code snippet in the middle makes the ordered list counter restart from `1`. 
I think it's best to explicitly specify the index `1.`, `2.`, etc...

File: README.md [hunk 2]
```diff
@@ -1,71 +1,152 @@
...
+```bash
+kustomize build . | kubectl apply -f -
+```
+
+### Common Customizations
+
+#### Disable constraints
+
+If you need to disable one of the constraints thats provided by default,
+you can do it by creating a kustomize patch like the following:
```
Comment (nikever) [inline README.md:96]: I would make it more concise:

_"Disable one of the default constraints by creating the following kustomize patch: "_

File: docs/COMPATIBILITY_MATRIX.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+| v1.3.1                              |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :warning:      |                    |                    |           |
+| v1.4.0                              |                    |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :warning:      |                    |           |
+| v1.5.0                              |                    |                    |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :warning:      |           |
+| v1.6.0                              |                    |                    |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :warning: |
+
+| Legend             | Description  |
+|:------------------:|--------------|
+| :white_check_mark: | Compatible   |
+| :warning:          | Has issues   |
+| :x:                | Incompatible |
```
Comment (nikever) [inline docs/COMPATIBILITY_MATRIX.md:22]: Don't think we put the legend in other matrices: https://github.com/sighupio/fury-kubernetes-networking/blob/master/docs/COMPATIBILITY_MATRIX.md


```
:white_check_mark: Compatible

:warning: Has issues

:x: Incompatible
```

File: katalog/gatekeeper/rules/README.md [hunk 4]
```diff
@@ -1,14 +1,62 @@
...
-- deny duplicated ingresses
-- unique service selector
-
-[security_controls_template.yml](templates/security_controls_template.yml): it's an all in one pod security policies
-measures, that contains a lot more rules, that are currently commented on. This file can be splitter into several
-`constraintTemplates` with the counterpart of having to manage an additional `constraint`/`constraintTemplate` resource
-for each one.
+<!-- <KFD-DOCS> -->
+
+The KFD OPA module provides out of the box a set of common polciies (rules) to get started in securing your cluster.
```
Comment (nikever) [inline katalog/gatekeeper/rules/README.md:5]: typo `polciies`

File: README.md [hunk 5]
```diff
@@ -1,71 +1,152 @@
...
-> You can comment out the service monitor in the [kustomization.yaml](./katalog/gatekeeper/core/kustomization.yaml)
-file if you don't want to install the monitoring module.
+## Packages
+
+Fury Kubernetes OPA provides the following packages:
+
+| Package                                             | Version  | Description                                                                      |
+|-----------------------------------------------------|----------|----------------------------------------------------------------------------------|
+| [Gatekeeper Core](katalog/gatekeeper/core)          | `v3.7.0` | Gatekeeper deployment, ready to apply rules.                                     |
+| [Gatekeeper Rules](katalog/gatekeeper/rules)        | `N.A.`   | A set of custom rules to get started. See the package documentation for details. |
```
Comment (nikever) [inline README.md:31]: I will remove "See the package documentation for details."

It is said after the table to click for more details

File: README.md [hunk 6]
```diff
@@ -1,71 +1,152 @@
...
-Kubernetes Cluster.
+![Release](https://img.shields.io/github/v/release/sighupio/fury-kubernetes-opa?label=Latest%20Release)
+![License](https://img.shields.io/github/license/sighupio/fury-kubernetes-opa?label=License)
+![Slack](https://img.shields.io/badge/slack-@kubernetes/fury-yellow.svg?logo=slack&label=Slack)
 
-## OPA Packages
+<!-- <KFD-DOCS> -->
 
-The following packages are included in the Fury Kubernetes OPA module:
+**Kubernetes Fury OPA** provides a policy engine based on OPA Gatekeeper to enable enforcement of custom policies in a Kubernetes Cluster for the [Kubernetes Fury Distribution (KFD)][kfd-repo] via a [Validating Admission Webhook][kubernetes-vaw-docs].
```
Comment (nikever) [inline README.md:12]: What do you think of simplifying it a little bit?

Something on the line of:

**Kubernetes Fury OPA** provides policy enforcement for the [Kubernetes Fury Distribution (KFD)][kfd-repo] using OPA Gatekeeper.

**Review by nikever — CHANGES_REQUESTED** [review]
I have some small comments here and there 🍯

File: README.md (hunk 2 above)
Comment (ralgozino) [reply README.md:96]: I like it!

File: README.md (hunk 1 above)
Comment (ralgozino) [reply README.md:73]: my vscode linter didn't like the 1. 2. ... but as you pointed out it is not working. fixed.

File: docs/COMPATIBILITY_MATRIX.md (hunk 3 above)
Comment (ralgozino) [reply docs/COMPATIBILITY_MATRIX.md:22]: fixed

File: README.md (hunk 6 above)
Comment (ralgozino) [reply README.md:12]: much better!

**Review by nikever — APPROVED** [review]

**Pull request merged**